js/piwik.js

```diff
@@ -2300,7 +2300,7 @@ if (typeof window.Piwik !== 'object') {
                 // detect this 100% correct for an iframe so whenever Piwik is loaded inside an iframe we presume
                 // the window had focus at least once.
                 hadWindowFocusAtLeastOnce = isInsideAnIframe(),
-                timeWindowLastFocussed = null,
```

Comment:
As this variable is only used locally and all uses are replaced, this should be fine (but the js files have to be regenerated)